sdk/storage/storage-blob/test/node/highlevel.node.spec.ts

```diff
@@ -764,4 +764,32 @@ describe("Highlevel", () => {
     });
     assert.equal((await blockBlobClient.getProperties()).accessTier, "Hot");
   });
+
+  it("uploadData should work with Buffer, ArrayBuffer and ArrayBufferView", async () => {
```

Comment:
Not testing the browser path for `uploadData`, please add!

Reply:
Done.